src/schemata/definitions/field_schema/quantity.yml

```
@@ -3,17 +3,49 @@ type: object
 description: |
   Defines a field of type `quantity`. A quantity is a real number with a unit
   represented as a string.
+
+  A quantity of a matching unit, but a different magnitude will be converted automatically.
 required: [type]
 properties:
   type:
     type: string
     enum: [quantity]
-  unit:  # TODO: Specify valid units.
+  unit:
     type: string
-    description: When set the value of this field should match the given unit.
-  kind:  # TODO: Specify valid kinds.
+    description: |
+      When set the value of this field should match the given unit.
+
+      Full list of supported units can be found [here](https://raw.githubusercontent.com/gentooboontoo/js-quantities/master/src/quantities/definitions.js)
```

```suggestion
      If a unit is specified, the value of the field should match that unit.
      
      See the list of [supported units](https://github.com/gentooboontoo/js-quantities/blob/master/src/quantities/definitions.js).
```
https://www.w3.org/QA/Tips/noClickHere